app/views/admin/pages/show.html.erb

```diff
@@ -1,11 +1,11 @@
 <form id="slices-form">
   <div id="container-actions">
-    <% if Slices::Translations.available? %>
+    <% if I18n.available_locales.many? %>
       <div class="toolbar translations">
         <nav>
           Edit in
           <ul>
-            <% Slices::Translations.all.each do |locale, language| %>
+            <% I18n.available_locales.each do |locale, language| %>
```

Comment:
@erino @calumgunn in case it wasn't obvious, I was looking for some feedback on the approach :-) Let me know what you think when you can.